src/table.rs

```diff
@@ -497,6 +520,8 @@ impl ToTokens for TableComponentDeriveInput {
                     // on_row_click(event);
                 };
 
+                let row_data_provider = items.clone();
```

Comment:
this doesn't seem to be used?

Reply:
It is not directly used in this function, but required to transfer the `items` to the `on_cell_change` closure in line `124`. I did not see a better way to do it.

Reply:
Oh yeah, seeing it now. 🤦🏻‍♂️ Sorry
I think it's ok for now, the component isn't prepared for large datasets yet anyway.